CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ include(CheckCXXCompilerFlag)
 
 set(MIOPEN_ENABLE_SQLITE On CACHE BOOL "")
 # Use SQLITE for compiled kernels, when turned off this will use raw files
-set(MIOPEN_ENABLE_SQLITE_KERN_CACHE On CACHE BOOL "")
+set(MIOPEN_ENABLE_SQLITE_KERN_CACHE Off CACHE BOOL "")
```

Comment:
This would disable a feature which is a customer requirement, if there is a bug we should fix it rather than disabling the feature. It is not clear to me from the linked comment whether we know for sure if the kernel cache is buggy.

Reply:
Yes, @JehandadKhan please create a branch and investigate. This issue is blocking our Jenkins CI. We need help on resolution. Disable all testing on your branch except the long tests and track this down.

Reply:
Does the CI pass if we disable this ?

Reply:
Yes. I started preparing this PR when my experimental branch passed 3 times out of 4 (only the "Full Long Tests" stage was used). Without this, I've often saw failures like shown in https://github.com/ROCmSoftwarePlatform/MIOpen/issues/226#issuecomment-633659969.